azure_arc_k8s_jumpstart/docs/local_kind.md

```diff
@@ -0,0 +1,160 @@
+
+    ```bash
+    az login
+    az ad sp create-for-rbac -n "<Unique SP Name>" --role contributor
+    ```
+
+    For example:
+
+    ```az ad sp create-for-rbac -n "http://AzureArcK8s" --role contributor```
+
+    Output should look like this:
+
+    ```
+    {
+    "appId": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "displayName": "AzureArcK8s",
+    "name": "http://AzureArcK8s",
+    "password": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "tenant": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+    ```
+    
+    **Note**: It is optional but highly recommended to scope the SP to a specific [Azure subscription and Resource Group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest) 
+
+* Enable subscription for two providers for Azure Arc enabled Kubernetes<br> 
+  Registration is an asynchronous process, and registration may take approximately 10 minutes.
+  ```bash
+  az provider register --namespace Microsoft.Kubernetes
+  Registering is still on-going. You can monitor using 'az provider show -n Microsoft.Kubernetes'
+
+  az provider register --namespace Microsoft.KubernetesConfiguration
+  Registering is still on-going. You can monitor using 'az provider show -n Microsoft.KubernetesConfiguration'
+  ```
+  You can monitor the registration process with the following commands:
+  ```bash
+  az provider show -n Microsoft.Kubernetes -o table
+ 
+  az provider show -n Microsoft.KubernetesConfiguration -o table
+  ```
+
+* Install the Azure Arc for Kubernetes CLI extensions ***connectedk8s*** and ***k8sconfiguration***:
+
+  ```bash
+  az extension add --name connectedk8s
+  az extension add --name k8sconfiguration
+  ```
+
+
+ 
+# Deployment
+
+* Install kind
+  * On Linux: 
+
+    ```bash
+    curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.9.0/kind-linux-amd64
+    chmod +x ./kind
+    sudo mv ./kind /usr/local/bin/kind
+    ```
+  * On Mac:
+
+    ```bash
+    brew install kind
+    ```
+  * On Windows:
+       ```powershell
+       choco install kind
+       ```
+
+
+* Navigate to the folder that has the kind cluster definition.
+  ```bash
+  cd azure_arc/azure_arc_k8s_jumpstart/kind
+  ```
+
+* Create the kind cluster
+    ```bash
+    kind create cluster --config kind_cluster.yaml
+    ```
+    ![](../img/kind_connect/01.png)
+
+* Verify your cluster was created successfully and you can access the cluster using `kubectl`.
+    ```bash
+    kubectl get nodes
+    ```
+    ![](../img/kind_connect/02.png)
+
+# Connecting to Azure Arc
+
+Now that you have a running kind cluster, lets connect the kind cluster to Azure Arc.
+  * Login to previously created [***Service Principal***](#prerequisites) <br>
+    ```bash
+    az login --service-principal -u mySpnClientId -p mySpnClientSecret --tenant myTenantID
+    ```
+
+ * Create a resource group<br> 
+   ```bash
+   az group create --name arckinddemo -l EastUS -o table
```

Comment:
```suggestion
   az group create --name Arc-kind-Demo -l EastUS -o table
```
This is to align with the rest of the repo. I am aware that there are some exceptions but this is the general naming convention.  
Also, update the screenshot accordingly.